tests/ui/simd/repr_packed.rs

```diff
@@ -0,0 +1,59 @@
+// run-pass
+
+#![feature(repr_simd, platform_intrinsics)]
+#![allow(non_camel_case_types)]
+
+#[repr(simd, packed)]
+struct Simd<T, const N: usize>([T; N]);
```

Comment:
From how `packed` usually works, I would expect this to mean that the type has alignment 1. But that doesn't seem to be the case, instead the alignment is the largest possible for the size, or something like that?

What happens with `packed(N)`?

Would be good to have the interaction of `simd` and `packed` documented somewhere.